asab/log.py

```diff
@@ -47,8 +47,15 @@ def __init__(self, app):
 			# Initialize file handler
 			file_path = Config["logging:file"]["path"]
```

Comment:
```
os.environ['HOSTNAME'] = app.HostName
```

... will do the job here, doesn't?

Reply:
@ateska I can skip the environment loading, so from:
${HOSTNAME}

this could be:
{HOSTNAME}

Hence, I do not have to modify the interpolation.

Reply:
no-one will understand it